Analyzer/test/condor/samples.py

```diff
@@ -24,8 +25,9 @@ def nSamples(self, name):
     def sampleList(self, name):
         files = self.lib.SC_samples(self.obj, name)
         names = self.lib.SC_samples_names(self.obj, name)
-        nEvts = self.lib.SC_samples_nEvts(self.obj, name)
-        list = [(files[i],names[i],nEvts[i]) for i in xrange(self.lib.SC_samples_size(self.obj, name))]
+        nGenEvts = self.lib.SC_samples_nGenEvts(self.obj, name)
+        nActEvts = self.lib.SC_samples_nActEvts(self.obj, name)
+        list = [(files[i],names[i],nActEvts[i]) for i in xrange(self.lib.SC_samples_size(self.obj, name))]
```

Comment:
Looks good to me. Seems like no need to change anything in condorSubmit.py

Reply:
Exactly, transparent to the user.